### PR TITLE
Deleting things to clean up the potree repo and make it more readable

### PR DESCRIPTION
**Pull request by calistenson**
All of this stuff looked deletable to me. Is it okay to remove everything in this PR? I want us to get to the point of separating our code from native potree code and having two separate repos; it'd be nice to move as little code as possible at that time.

### CODE REVIEW

File: data-labeling/inputform.js [hunk 1]
```diff
@@ -42,26 +42,6 @@ $(document).ready(function() {
     }
 
     function fillForm() {
-      // var valid = true;
```
Comment (vrajur-nd) [inline data-labeling/inputform.js]: leave this

File: common/playbar.js [hunk 2]
```diff
@@ -167,40 +154,22 @@ $(document).ready(function () {
...
     });
 
     playbarhtml.find("#pausebutton").mousedown(function() {
-      console.log("PAUSE");
       playbarhtml.find("#playbutton").show();
       playbarhtml.find("#pausebutton").hide();
 
     });
 
-    window.addEventListener("keypress", (e) => {
```
Comment (vrajur-nd) [inline common/playbar.js]: leave this

File: demo/basemap-time.html [hunk 3]
```diff
@@ -113,7 +113,6 @@
```
Comment (vrajur-nd) [inline demo/basemap-time.html]: This whole file can actually be deleted

File: demo/radar.html [hunk 4]
```diff
@@ -695,27 +670,6 @@
 	});
 });
 
-	// let gapShaderMaterial = shaderMaterial.clone();
```
Comment (vrajur-nd) [inline demo/radar.html]: leave this

File: common/playbar.js (hunk 2 above)
Comment (calistenson) [reply common/playbar.js]: Going to delete it and track reinstating it in this Zenhub issue: 
https://app.zenhub.com/workspaces/veritas-5b9aa680dfeccd3433cd6f62/issues/nextdroid/potree/24

File: demo/radar.html (hunk 4 above)
Comment (calistenson) [reply demo/radar.html]: Going to delete and track in this issue: https://app.zenhub.com/workspaces/veritas-5b9aa680dfeccd3433cd6f62/issues/nextdroid/potree/25

**Pull request merged**